package.json

```diff
@@ -10,7 +10,9 @@
     "flow": "flow check",
     "eslint": "eslint ./src -c ./.eslintrc.yml --cache --cache-location ./.cache/.eslintcache",
     "test": "jest --config=./jest.config.js --forceExit --detectOpenHandles --no-cache",
-    "ci:check": "concurrently 'yarn test' 'yarn eslint' 'yarn flow' --kill-others-on-fail"
+    "ci:check": "concurrently 'npm run test' 'npm run eslint' 'npm run flow' --kill-others-on-fail",
```

Comment:
What is the reason for the change?

Reply:
saves a little bit of time on Travis CI, because we don't have to install yarn, and I don't think it has any negative effects

Reply:
Ok, cool :)